web/src/main/java/io/openshift/appdev/missioncontrol/web/api/ValidationResource.java

```diff
@@ -118,6 +121,23 @@ public Response openShiftTokenExists(@HeaderParam(HttpHeaders.AUTHORIZATION) fin
         }
     }
 
+    private Identity getIdentity(String authorization, String cluster) {
```

Comment:
I think there is a similar method in the parent class? Maybe worth naming getOpenShiftIdentity

Reply:
the name of the method in parent is called `getDefaultOpenShiftIdentity` now the method is called the same as the one in keycloak service